pkg/performanceprofile/controller/performanceprofile/hypershift/hypershift.go

```diff
@@ -72,13 +73,25 @@ func (ci *ControlPlaneClientImpl) getFromConfigMap(ctx context.Context, key clie
 	if err != nil {
 		return err
 	}
+
+	cmKey := client.ObjectKeyFromObject(cm)
 	var objAsYAML string
-	// can't have both
+	var tuningKeyFound, configKeyFound bool
 	if s, ok := cm.Data[TuningKey]; ok {
 		objAsYAML = s
+		tuningKeyFound = true
 	}
-	if s, ok := cm.Data[ConfigKey]; ok {
+	if s, ok2 := cm.Data[ConfigKey]; ok2 {
 		objAsYAML = s
+		configKeyFound = true
+	}
+	// can't have both
```

Comment:
protection added due to comment here: https://github.com/openshift/cluster-node-tuning-operator/pull/1004#discussion_r1604596493